src/ApiPort/ApiPort.VisualStudio.Common/ComProjectMapper.cs

```diff
@@ -16,7 +16,7 @@
 namespace ApiPortVS
 {
     /// <summary>
-    /// You need to switch to the UI thread before calling any COM interfaces (e.g. IVsHierarchy) using
+    /// You need to switch to the UI thread before calling any COM interfaces (e.g. IVsHierarchy) using.
```

Comment:
It feels like maybe there used to be another line here so that it said something like "...using ThreadHelper.JoinableTaskFactory.SwitchToMainThreadAsync()".  As is, it doesn't make sense, and the period doesn't help.

Reply:
Fixed by PR https://github.com/microsoft/dotnet-apiport/pull/811